test/XRP/xrp-protocol-buffer-conversion.test.ts

```diff
@@ -74,6 +76,30 @@ describe('Protocol Buffer Conversion', function (): void {
     assert.deepEqual(currency.name, testCurrencyProto.getName())
   })
 
+  it('Convert Currency protobuf missing required field name', function (): void {
+    // GIVEN a Currency protocol buffer missing a name.
+    // WHEN the protocol buffer is converted to a native Typescript type THEN an error is thrown.
```

Comment:
is it a common pattern to have THEN on the same line as WHEN if it fits? on first glance i thought it was missing

Reply:
I've seen it in a lot of other tests - I don't think I've seen them on separate lines when they're right next to each other

Reply:
dope, thanks!